src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int defaultCapacity = 16;
+    public static final double loadFactor = 0.75;
+    public static final int multiplier = 2;
```

Comment:
The constant 'multiplier' should be named in uppercase as 'MULTIPLIER' to follow Java naming conventions for constants.